templates/data-protector-Backup/dpForUnix.cfg

```diff
@@ -0,0 +1,1142 @@
+PATROL_CONFIG
+"/SENTRY/X/dpForUnix/actionList" = { REPLACE = "" },
+"/SENTRY/X/dpForUnix/credentialsList" = { REPLACE = "" },
+"/SENTRY/X/dpForUnix/databaseType" = { REPLACE = "" },
+"/SENTRY/X/dpForUnix/description" = { REPLACE = "Micro Focus Data Protector Template for Linux/UNIX" },
```

Comment:
Please add the required macros in the description, so people know what to put in there:

```md
- **DP_BIN_PATH**: Data Protector binary path. By default this points to %DP_HOME_DIR%/bin on Windows and /opt/omni/bin on Linux/UNIX
- **CLIENT_HOURS**: Number of hours client statistics are searched for. Default: 72 hours
- **SESSION_HOURS**: Number of hours sessions are monitored for. Default: 72 hours
```

Reply:
Fixed.